packages/cisco_aironet/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,302 @@
+---
+description: Pipeline for processing Cisco Aironet Logs
+processors:
+  - rename:
+      field: message
+      target_field: event.original
+      ignore_missing: true
+      ignore_failure: true
+  - set:
+      field: ecs.version
+      value: '8.3.1'
+  - grok:
+      field: event.original
+      patterns:
+        - "%{SYSLOG_HEADER}:\\s%%{GREEDYDATA:_temp_.full_message}"
+        - "%{SYSLOGFACILITY}%{INT}: AP:%{MAC:host.mac}: \\*%{AIRONET_DATE:_temp_.raw_date}: %%{GREEDYDATA:_temp_.full_message}"
+        - "%{SYSLOGFACILITY}%{DATA:host.name}: -%{GREEDYDATA:_temp_.full_message}"
+      pattern_definitions:
+        SYSLOG_HEADER: "%{SYSLOGFACILITY}%{DATA:host.name}:\\s\\*%{DATA:process.name}:\\s%{AIRONET_DATE:_temp_.raw_date}"
+        SYSLOGFACILITY: "<%{NONNEGINT:log.syslog.priority:int}>"
```

The old naming of this was incorrect (my error), let's clean that up in follow-up PRs.
```suggestion
        - "%{SYSLOGFACILITY}%{INT}: AP:%{MAC:host.mac}: \\*%{AIRONET_DATE:_temp_.raw_date}: %%{GREEDYDATA:_temp_.full_message}"
        - "%{SYSLOGPRIORITY}%{DATA:host.name}: -%{GREEDYDATA:_temp_.full_message}"
      pattern_definitions:
        SYSLOG_HEADER: "%{SYSLOGPRIORITY}%{DATA:host.name}:\\s\\*%{DATA:process.name}:\\s%{AIRONET_DATE:_temp_.raw_date}"
        SYSLOGPRIORITY: "<%{NONNEGINT:log.syslog.priority:int}>"
```